hotshot-state-prover/src/circuit.rs

```diff
@@ -439,24 +448,26 @@ mod tests {
             .map(|b| if b { F::from(1u64) } else { F::from(0u64) })
             .collect::<Vec<_>>();
         // good path
-        let (circuit, public_inputs) = build::<_, _, _, _, _, ST_CAPACITY>(
+        let (circuit, public_inputs) = build::<_, _, _, _, _>(
```

Comment:
```suggestion
        let (circuit, public_inputs) = build(
```

one benefit of removing that const generic is to remove all the generic type hint. 
same comment applied to everywhere else where we still have dangling `<_, _, ...>`